sync_known_issues.py

```diff
@@ -284,35 +276,35 @@ def main():
                                 known_issue.title))
                         affected_environments.append(api_env)
 
-            if not args.dry_run:
-                known_issue_api_object = {
-                    'title': known_issue.title,
-                    'test_name': known_issue.test_name,
-                    'url': known_issue.url,
-                    'notes': known_issue.notes,
-                    'active': known_issue.active,
-                    'intermittent': known_issue.intermittent,
-                    'environment': [item['url'] for item in affected_environments]
-                }
-
-                if api_known_issue is None:
+            known_issue_api_object = {
+                'title': known_issue.title,
+                'test_name': known_issue.test_name,
+                'url': known_issue.url,
+                'notes': known_issue.notes,
+                'active': known_issue.active,
+                'intermittent': known_issue.intermittent,
+                'environment': [item['url'] for item in affected_environments]
+            }
+
+            if api_known_issue is None:
+                print("Adding issue:")
```

Comment:
use logging.info ?

Reply:
I'll try and see how it's formatted. Sometimes print() is nice for mandatory things (as in, actual purposeful output).